types/twilio-video/index.d.ts

```diff
@@ -170,6 +171,38 @@ export class MediaServerRemoteDescFailedError extends TwilioError {
     code: 53403;
     message: 'Server is unable to apply a remote media description';
 }
+export class NetworkQualityAudioStats extends NetworkQualityMediaStats { }
+export class NetworkQualityBandwidthStats {
+    actual: number | null;
+    available: number | null;
+    level: NetworkQualityLevel;
+}
+export class NetworkQualityFractionLostStats {
+    fractionLost: number | null;
+    level: NetworkQualityLevel;
+}
+export class NetworkQualityLatencyStats {
+    jitter: number | null;
+    rtt: number | null;
+    level: NetworkQualityLevel;
```

Comment:
```suggestion
    level: NetworkQualityLevel | null;
```
https://media.twiliocdn.com/sdk/js/video/releases/2.0.0-beta14/docs/NetworkQualityLatencyStats.html